packages/gatsby-mdx/on-create-node.js

```diff
@@ -37,17 +36,11 @@ module.exports = async (
 
   // extract all the exports
   const nodeExports = extractExports(code);
+  const { frontmatter } = nodeExports;
```

Comment:
would need to make this change here too -- https://github.com/ChristopherBiscardi/gatsby-mdx/blob/master/packages/gatsby-mdx/utils/create-mdx-node.js